src/app/delegates/components/IssuesFilter.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+
+import { useRouter, useSearchParams } from "next/navigation";
+import { Listbox } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDown } from "lucide-react";
+import { useAddSearchParam, useDeleteSearchParam } from "@/hooks";
+import { useAgoraContext } from "@/contexts/AgoraContext";
+import Tenant from "@/lib/tenant/tenant";
+
+const FILTER_PARAM = "issueFilter";
+const DEFAULT_FILTER = "all";
+
+export default function IssuesFilter() {
+  const { ui } = Tenant.current();
+
+  const router = useRouter();
+  const searchParams = useSearchParams();
+  const addSearchParam = useAddSearchParam();
+  const deleteSearchParam = useDeleteSearchParam();
+  const filterParam = searchParams?.get(FILTER_PARAM) || "all";
+  const { setIsDelegatesFiltering } = useAgoraContext();
+
+  const hasIssues = Boolean(
+    ui.governanceIssues && ui.governanceIssues.length > 0
+  );
+  if (!hasIssues) return null;
+
+  let issuesFilterOptions: any = {
+    [DEFAULT_FILTER]: {
+      value: "All Issues",
+      sort: DEFAULT_FILTER,
+    },
+  };
+
+  // Map values to sort options
+  ui.governanceIssues!.forEach((item) => {
+    issuesFilterOptions[item.key] = {
+      value: item.title,
+      sort: item.key,
+    };
+  });
+
+  const handleChange = (value: string) => {
+    setIsDelegatesFiltering(true);
+    router.push(
+      value === DEFAULT_FILTER
+        ? deleteSearchParam({ name: FILTER_PARAM })
+        : addSearchParam({ name: FILTER_PARAM, value }),
+      { scroll: false }
+    );
+  };
+
+  return (
+    <Listbox
+      as="div"
+      value={filterParam || issuesFilterOptions.all.value}
+      onChange={(value) => handleChange(value)}
+    >
+      {() => (
+        <>
+          <Listbox.Button className="w-full sm:w-fit bg-[#F7F7F7] text-base font-medium border-none rounded-full py-2 px-4 flex items-center">
```

Comment:
some of these `bg-[#...]` color things are why I want to merge the theme branch asap so we can get everyone on the same page using `bg-primary` or whatever it may be from the theme presets

We can merge this first and I can rebase, not a huge problem. But want to get my theme branch in so we don't have to catch up all of the new features last minute